lib/player.js

```diff
@@ -685,6 +703,68 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
     if (mediaElement) {
       this.attach(mediaElement, /* initializeMediaSource= */ true);
     }
+
+    if (canvas) {
+      this.canvas_ = canvas;
+    }
+  }
+
+  /**
+   * Create a shaka.lcevc.Dil object
+   * @private
+   */
+  createDIL_() {
+    if (this.lcevcDil_ == null) {
+      this.lcevcDil_ = new shaka.lcevc.Dil(
+          /** @type {HTMLVideoElement} */ (this.video_),
+          this.canvas_,
+          this.lcevcDilConfig_,
+      );
+      if (this.mediaSourceEngine_) {
+        this.mediaSourceEngine_.updateLcevcDil(this.lcevcDil_);
+      }
+    }
+  }
+
+  /**
+   * Close a shaka.lcevc.Dil object
```

Comment:
You should mention this also hides the canvas. Could be a surprise, for a user-defined canvas.

Reply:
changed to `Close a shaka.lcevc.Dil object if present and hide the canvas.`